zynqmp/optee-os/files/0001-allow-setting-sysroot-for-libgcc-lookup.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/mk/gcc.mk b/mk/gcc.mk
```

Comment:
Same here, I'd like to avoid doing live patching if possible.

Reply:
patch removed, workaround provided by @etienne-lms